crates/rust-eigenda-v2-client/src/cert_verifier.rs

```diff
@@ -19,11 +20,12 @@ pub struct CertVerifier {
 
 impl CertVerifier {
     /// Creates a new instance of CertVerifier receiving the address of the contract and the ETH RPC url.
-    pub fn new(address: String, rpc_url: String) -> Self {
+    pub fn new(address: H160, rpc_url: String) -> Self {
```

Comment:
this function should return a Result, since the rpc_url is of type String (it can be an empty-string or something that is not an Url)